docs/architecture/shared-nothing.txt

```diff
@@ -125,39 +125,44 @@ cluster and coordinate operations:
 * Schemas of tables
 * The status and location of primary and replica shards
 
-When the Primary updates the cluster state it will publish the new state to all
+When the master node updates the cluster state it will publish the new state to all
 nodes in the cluster and and wait for all nodes to respond before processing
 the next update.
 
 .. _shared_nothing_primary_election:
 
-Primary Election
-----------------
+Master Node Election
+--------------------
 
-Shared nothing architecture is often also referred to as "masterless"
-architecture. For CrateDB, this is fully true in terms of data storage.
-However, coordinating metadata, settings and DDL requires to have a dynamic
-Metadata Primary to be elected. In a CrateDB cluster any node is eligible to be
-elected as a Metadata Primary, although this could also be restricted to a
-subset of nodes if required. There must be only one single Metadata Primary per
-cluster.
+The process of designating a single node as master node in a cluster is
+called Master Node Election. There must be only one single master node per cluster.
+In a CrateDB cluster any node is eligible to be elected as a master node, although
+this could also be restricted to a subset of nodes if required.
+These nodes will then elect a single node as master to coordinate the cluster state
+across the cluster.
 
-To ensure that even in case of a network partition (when one part of the
-cluster is not available) the cluster requires a quorum (relative to the
-expected nodes in the cluster) to be able to elect the Metadata Primary.
+
+.. NOTE::
+   The following setting is only important for all 3 versions of CrateDB.
```

Comment:
I don't understand this note, can you please elaborate what you want to express here?